index.js

```diff
@@ -21896,6 +21926,7 @@ function BeeSwarmSimulator(DATA){
             out.ownsCobaltBee=false
             out.cloudBoostAmount=1.15
             out.beeColorAmounts={r:0,b:0,w:0}
+            out.beeRarityAmounts={c:0,r:0,e:0,l:0,m:0}

```

Comment:
If anything breaks, it is probably because this line is adding another parameter to the state without explicitly accounting for storing that state in the save code.